assets/MicrosoftUpdate/RegisterMicrosoftUpdate.ps1

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+param(
+    [ValidateSet('Registration', 'Sleep', 'Fail')]
```

Comment:
```suggestion
    [ValidateSet('Hang', 'Fail')]
```

Reply:
I remember we removed `Hang` term from our C# file for compliance. Can we use the term here?